packages/react/src/hooks/utils/query/useQueryClient.ts

```diff
@@ -0,0 +1,5 @@
+import { useQueryClient as rqUseQueryClient } from '@tanstack/react-query'
```

Comment:
Kinda prefer aliasing to `RQ_useQueryClient`. @jxom what do you think?

Reply:
I don't mind changing it.  Personally I prefer sticking to camelCase and not mixing.

Reply:
Could also do `useQueryClient as useQueryClient_` – that is the pattern we use for some dupe import stuff atm.